cmake/extract_includes.bat.in

```diff
@@ -0,0 +1,144 @@
+mkdir include
```

Comment:
@mkruskal-google deleted this file in #10306, you should sync with him on whether to sync this back in/how to delete it from our sync process.

Reply:
Yea I wonder why it didn't sync the deletion back to google3 in Sandy's sync?  We should delete this here and in google3

Reply:
Done.